src/pages/PeoplePage/PeopleFilters.tsx

```diff
@@ -0,0 +1,122 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SearchLink } from '../../utils/SearchLink';
+
+const centuriesValues = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters = () => {
+  const [searchParams, setSarchParams] = useSearchParams();
+  const sex = searchParams.get('sex');
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries');
+
+  const quaryUpdateHandler = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setSarchParams(
+      getSearchWith(searchParams, { query: e.target.value || null }),
+    );
+  };
+
+  const SexFilterLinks = (sexType: string | null) => [
+    {
+      name: 'All',
```

Comment:
Create enum for this
```suggestion
      name: SexFilterEnum.All,
```